packages/eas-json/schema/eas.schema.json

```diff
@@ -311,7 +342,28 @@
                 "macos-big-sur-11.4-xcode-13.0",
                 "macos-big-sur-11.4-xcode-12.5"
               ],
-              "markdownEnumDescriptions": ["`macos-monterey-12.6-xcode-14.0`", "`macos-monterey-12.6-xcode-14.1`"]
+              "markdownEnumDescriptions": [
+                "`macos-monterey-12.6-xcode-14.0`",
+                "`macos-monterey-12.6-xcode-14.1`"
+              ]
+            },
+            {
+              "type": "string"
+            }
+          ]
+        },
+        "resourceClass": {
+          "description": "The iOS-specific resource class that will be used to run this build. Learn more: https://docs.expo.dev/build-reference/infrastructure/",
+          "markdownDescription": "The iOS-specific resource class that will be used to run this build. [Learn more](https://docs.expo.dev/build-reference/infrastructure/)",
+          "default": "default",
+          "anyOf": [
+            {
+              "enum": ["default", "m1-medium", "intel-medium"],
+              "markdownEnumDescriptions": [
+                "Medium Intel resource class",
+                "Medium M1 resource class",
+                "Medium Intel resource class"
+              ]
```

Comment:
Should I add a `large` option here? I skipped it because in the next PR, I will make WWW use medium Intel instance for requestedResourceClass = LARGE.

Reply:
Can you elaborate? (on Slack)